logic/transaction_pool.js

```diff
@@ -298,20 +298,29 @@ TransactionPool.prototype.getMergedTransactionList = function(reverse, limit) {
  * @param {Object} transaction - Transaction object
  */
 TransactionPool.prototype.addUnconfirmedTransaction = function(transaction) {
+	const that = self;
```

Comment:
Can ```const that = self``` be removed? Is there any advantage using ```const that = self``` in this function instead of using self?